relay/p2p/broadcaster.go

```diff
@@ -135,26 +147,32 @@ func (b *broadcaster) ProcessReceivedMessage(message p2p.MessageP2P, _ core.Peer
 	b.log.Debug("got message", "topic", message.Topic(),
 		"msg.Payload", msg.Payload, "msg.Nonce", msg.Nonce, "msg.PublicKey", addr.AddressAsBech32String())
 
+	err = b.processNonce(msg)
+	if err != nil {
+		// someone might try to send old, already seen by the network, messages
+		// drop the message and do not resend-it to other relayers
+		return err
+	}
+
 	switch message.Topic() {
-	case joinTopicName:
-		b.processJoinMessage(msg, message)
-	case signTopicName:
+	case b.name + joinTopicSuffix:
```

Comment:
since `b.name` isn't likely to change and joinTopicSuffix is a constant, why not store these in the struct as members, as not to calculate them every single time?
also for `b.name + signTopicSuffix`

Reply:
changed